### PR TITLE
Updating Safari TP and Webkit release info and results

### PR DESCRIPTION
**Pull request by afmenez**
Updating Safari TP and Webkit release info and results according to the release announcement:
https://webkit.org/blog/7922/release-notes-for-safari-technology-preview-40/

I haven't run any tests.

### CODE REVIEW

**Comment by afmenez** [comment]
Updating Safari TP and Webkit release info again, according to the release announcement:
https://webkit.org/blog/7989/release-notes-for-safari-technology-preview-41/

There are no JS changes on this release.  I haven't run any tests.

**Review by ljharb — APPROVED** [review]

File: data-esnext.js [hunk 1]
```diff
@@ -1719,6 +1719,8 @@ exports.tests = [
     ie11: false,
     firefox2: false,
     chrome61: chrome.harmony,
+    safari1: false,
```
Comment (afmenez) [inline data-esnext.js]: actually, this was supposed to be safari11, but i guess it is correct anyway.

Comment (ljharb) [reply data-esnext.js]: might as well update it :-)

Comment (afmenez) [reply data-esnext.js]: Actually, i'll remove this line, since I haven't actually tested it.

Comment (afmenez) [reply data-esnext.js]: Hum, I found a lot of safari1 results that I believe I added previously.  I'll remove them too.

Comment (afmenez) [reply data-esnext.js]: Should I remove previous safaritp and webkit results too?  I mean, the results refer to a previous version (not TP41).  I would obviously only remove when the results are the same as Safari 11.

Comment (ljharb) [reply data-esnext.js]: I'd say we should be keeping only results that were explicitly tested.

**Comment by afmenez** [comment]
I don't understand why PR #1193, which updates the results without updating the header, was merged and this one was not.  Anyway, I rebased the the patch and now no results are changed, just the header.  i still believe it needs merging.

**Review by chicoxyzzy — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
Sorry. Merged.

**Comment by afmenez** [comment]
Tks!

**Pull request merged**